returnn/tf/layers/signal_processing.py

```diff
@@ -270,111 +325,106 @@ def tfMelFilterBank(fMin, fMax, samplingRate, fftSize, nrOfFilters):
       The filter matrix is computed according to equation 6.141 in
       [Huang & Acero+, 2001] "Spoken Language Processing - A Guide to Theroy, Algorithm, and System Development"
 
-      :type fMin: float | int
-      :param fMin: minimum frequency
-      :type fMax: float | int
-      :param fMax: maximum frequency
-      :type samplingRate: float
-      :param samplingRate: sampling rate of audio signal
-      :type fftSize: int
-      :param fftSize: dimension of discrete fourier transformation
-      :type nrOfFilters: int
-      :param nrOfFilters: number of mel frequency filter banks to be created
-
+      :param float|int fMin: minimum frequency
+      :param float|int fMax: maximum frequency
+      :param float samplingRate: sampling rate of audio signal
+      :param int fftSize: dimension of discrete fourier transformation
+      :param int nrOfFilters: number of mel frequency filter banks to be created
       :rtype: tf.tensor, shape=(filterValue, nrOfFilters)
       :return: matrix yielding the mel frequency cepstral coefficients
       """
       import numpy as np
 
-      def melScale(freq):
+      def mel_scale(freq):
         """
         returns the respective value on the mel scale
 
-        :type freq: float
-        :param freq: frequency value to transform onto mel scale
+        :param float freq: frequency value to transform onto mel scale
         :rtype: float
         """
         return 1125.0 * np.log(1 + float(freq) / 700)
 
-      def invMelScale(melVal):
+      # noinspection PyPep8Naming
+      def inv_mel_scale(melVal):
         """
         returns the respective value in the frequency domain
 
-        :type melVal: float
-        :param melVal: value in mel domain
+        :param float melVal: value in mel domain
         :rtype: float
         """
         return 700.0 * (np.exp(float(melVal) / 1125) - 1)
 
-      def filterCenter(filterId, fMin, fMax, samplingRate, fftSize, nrOfFilters):
+      # noinspection PyPep8Naming,PyShadowingNames
```

Comment:
Why `PyPep8Naming`? You should only ignore this warning for very good reasons. I don't understand why it is needed here. The arguments should be renamed to match the PEP8 style.

Reply:
I did rename the arguments previously but you commented:
> In the case that you need to rename arguments of the local function, I would rather prefer to not do that. So the `# noinspection ...` is a cleaner here, I think.

Therefore, I undid the renaming and used `# noinspection PyPep8Naming` instead

Reply:
Sorry, this was about `PyShadowingNames`. You do not necessarily need to (or should not) rename arguments to avoid `PyShadowingNames`.

You almost always should avoid `PyPep8Naming`.

Reply:
Okay, then I'll rename the arguments again. 

Is there some guideline or way of determining these priorities?

Reply:
Mostly just common sense. :)

Reply:
Sure. As I said, I renamed them in the first place and just undid it because of your comment.

Reply:
If I rename them anyway, should we then also avoid `PyShadowingNames` warnings or still use `noinspection` there?

Reply:
As I said (that is why I meant with my initial comment), I would usually not rename the arguments just for `PyShadowingNames`.